nbs/tutorial_colab.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>Clean up the language here</p><p> </p><p><s>to be worked out of that</s> performs the following setup steps:</p><p> </p><ol><li>Mount Google Drive</li><li>Change your libraries directory</li><li>....</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>